src/Bicep.Decompiler/BicepDecompiler.cs

```diff
@@ -111,12 +131,19 @@ public async Task<DecompileResult> Decompile(Uri entryJsonUri, Uri entryBicepUri
                 continue;
             }
 
-            filesToSave[fileUri] = PrettyPrinter.PrintProgram(bicepFile.ProgramSyntax, new PrettyPrintOptions(NewlineOption.LF, IndentKindOption.Space, 2, false));
+            filesToSave[fileUri] = PrettyPrinter.PrintProgram(bicepFile.ProgramSyntax, GetPrettyPrintOptions());
         }
 
         return filesToSave.ToImmutableDictionary();
     }
 
+    private static string PrintSyntax(SyntaxBase syntax)
+    {
+        return PrettyPrinter.PrintSyntax(syntax, GetPrettyPrintOptions());
+    }
+
+    private static PrettyPrintOptions GetPrettyPrintOptions() => new PrettyPrintOptions(NewlineOption.LF, IndentKindOption.Space, 2, false);
```

Comment:
Worth defining this as a default somewhere (e.g. `PrettyPrintOptions.Default`)? It's repeated quite a bit in the code elsewhere.

Reply:
Will explore this in a separate PR as well.  The current default for newline option is inconsistent so I'd like separate views on the change (and hopefully make them consistent - i.e., Auto vs LF).